R/util.r

```diff
@@ -116,10 +116,18 @@ parse_period_unit <- function(unit) {
 
 date_to_posix <- function(date, tz = "UTC") {
   utc <- .POSIXct(unclass(date) * 86400, tz = "UTC")
-  if (tz == "UTC") utc
+  if (is_utc(tz)) utc
   else force_tz(utc, tz)
 }
 
+# UTC-equivalent timezones can be treated as UTC;
+#   check grep('UTC|GMT', OlsonNames(), value = TRUE)
+is_utc = function(tz) {
+  utc_tz = c("UTC", "GMT", "Etc/UTC", "Etc/GMT", "GMT-0", "GMT+0", "GMT0")
+  if (is.null(tz)) tz = Sys.timezone()
```

Comment:
Why is this NULL check?

Reply:
this is for input from Sys.time() or otherwise using default timezone